src/renderers/webgl/WebGLTextures.js

```diff
@@ -916,12 +940,16 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 		} else if ( renderTarget.depthBuffer && renderTarget.stencilBuffer ) {
 
-			if ( isMultisample ) {
+			const samples = getRenderTargetSamples( renderTarget );
```

Comment:
You're now calling getRenderTargetSamples even in the non-multisampled case. Does that matter? What happens if you call getRenderTargetSamples on a non-multisampled target? Is there any perf overhead for doing this if you don't need the samples?

Reply:
No, this function just reads out a predefined value. It doesn't call into GL,